### PR TITLE
Changed text in home page

### PR DESCRIPTION
**Pull request by DhrutikPatel**
Issue: #1109

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,7 +167,7 @@ <h1 class="heading"><i class="fas fa-cloud"></i> sButton CDN</h1>
         <div class="instruction row">
           <div class="desc col col-md-6 col-sm-12">
             When you only need to include sButtons’s compiled CSS , you can use
-            SButtonCDN.
+            SButton CDN.
```
Comment (shahednasser) [inline index.html:170]: Change SButton to sButtons and a link to it.

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (DhrutikPatel) [reply index.html:170]: I have to add this link - ("https://cdn.jsdelivr.net/gh/sbuttons/sbuttons/dist/sbuttons.css") to "sButton CDN" in heading or in subsection ?

Comment (shahednasser) [reply index.html:170]: in the subsection and in the code block

Comment (DhrutikPatel) [reply index.html:170]: ok

**Comment by shahednasser** [comment]
Awesome work thank you!

@all-contributors please add @DhrutikPatel for content

**Pull request merged**